client/my-sites/email/email-management/home/email-list-inactive.jsx

```diff
@@ -1,18 +1,54 @@
 import { Button, Card } from '@automattic/components';
 import { localize } from 'i18n-calypso';
 import React from 'react';
+import emailIllustration from 'calypso/assets/images/email-providers/email-illustration.svg';
+import PromoCard from 'calypso/components/promo-section/promo-card';
 import SectionHeader from 'calypso/components/section-header';
 import { canCurrentUserAddEmail } from 'calypso/lib/domains';
-import { emailManagementPurchaseNewEmailAccount } from 'calypso/my-sites/email/paths';
+import {
+	emailManagementPurchaseNewEmailAccount,
+	isUnderEmailInbox,
+} from 'calypso/my-sites/email/paths';
 
 class EmailListInactive extends React.Component {
+	getMainHeader() {
+		const { translate } = this.props;
+		const image = {
+			path: emailIllustration,
+			align: 'right',
+		};
+
+		return (
+			<>
+				<PromoCard
+					isPrimary={ true }
```

Comment:
We don't need to specify `true` here, it can be omitted (this is the default value for props not specified):

```suggestion
					isPrimary
```

Reply:
Actually it was needed :D default value was undefined.

Reply:
`PromoCard` is a TypeScript component, maybe that's why. I had a look to the documentation, and interestingly it is now [discouraged to omit that value](https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true). Good to know!